common/interfaces/desktop_app_interface/communication.c

```diff
@@ -420,7 +424,7 @@ static void comm_packet_parser(const uint8_t *data, const uint16_t length) {
     static uint32_t packet_crc = 0;
 
     if (memcmp(data, SDK_REQ_PACKET, CY_MIN(sizeof(SDK_REQ_PACKET), length)) == 0)
-        return lusb_write(SDK_RESP_PACKET, sizeof(SDK_RESP_PACKET));
+        return BSP_DebugPort_Write(SDK_RESP_PACKET, sizeof(SDK_RESP_PACKET));
```

Comment:
The function `BSP_DebugPortWrite` should not be used for USB write purposes.

Either change the function name from `BSP_DebugPort_Write` to `usb_write` or use build switches to use
`lusb_write` for stm32 and `SIM_Transmit_FS` for simulator

Reply:
changed. separated using build switch now.